index.html

```diff
@@ -13,7 +13,20 @@

```

Comment:
this image link is not acceptable

Reply:
> this image
![Screenshot from 2023-10-04 12-21-03](https://github.com/italanta/elewa-website-html/assets/102291617/2255cf03-0e2e-488c-8d90-ea6c4349d939)
ge link is not acceptable
![Screenshot from 2023-10-03 21-40-05](https://github.com/italanta/elewa-website-html/assets/102291617/02f310d2-d407-49ed-91b8-2d02db5acc13)